mmv1/provider/terraform.go

```diff
@@ -365,7 +365,14 @@ func (t Terraform) CopyFileList(outputFolder string, files map[string]string, ge
 			log.Fatalf("Cannot read source file %s while copying: %s", source, err)
 		}
 
-		err = os.WriteFile(targetFile, sourceByte, 0644)
+		var permission fs.FileMode
+		if strings.HasSuffix(targetDir, "scripts") {
+			permission = 0755
```

Comment:
This resolves a diff for the script files that Github picks up on. May as well remove that diff.